Makefile

```diff
@@ -210,6 +239,16 @@ check_bear_installed:
 		echo >&2 "bear is not installed. Please install bear to continue."; exit 1; \
 	}
 
+# Create coverage report to display with coverage gutter
+.PHONY: coverage
+coverage: | $(COV_DIR)
+	@printf "$(YELLOW)$(BOLD)Creating coverage report$(RESET) [$(BLUE)$@$(RESET)]\n"
+	$(SILENT)kcov --cobertura-only $(COV_DIR) ./$(TEST)
```

Comment:
I added it on an extra line, and put the coverage report in the same COV_DIR. 

We could host coverage data on something like https://coveralls.io/ but I think this would be better to do per Github action. So I don't think there is a disadvantage having cov.xml and index.html in the same COV_DIR 🤔 

Coverage Gutter has a Live Preview functionality. The search for coverage data is limited to COV_DIR by design: this should make it a bit faster, since the extension doesn't have to travel the whole project.
Now the target creates cov.xml and also the normal reports with index.html.
You can launch a preview it with Ctrl+Shift+6

Also excluded the system headers and our testfiles - which made the coverage drop significantly :o 

Also added COV_DIR to fclean.

Reply:
Great, aside from the coverage drop :(

I don't see any issue with putting everything in the coverage dir. My initial idea was to create a separate dir for organizational purposes, but it's not strictly necessary.